services/slack/intermediate.go

```diff
@@ -108,16 +108,16 @@ func (u *IntermediateUser) Sanitise(logger log.FieldLogger, defaultEmailDomain s
 }
 
 type IntermediatePost struct {
-	User     string                `json:"user"`
-	Channel  string                `json:"channel"`
-	Message  string                `json:"message"`
-	Props    model.StringInterface `json:"props"`
-	CreateAt int64                 `json:"create_at"`
-	// Type           string              `json:"type"`
-	Attachments    []string            `json:"attachments"`
-	Replies        []*IntermediatePost `json:"replies"`
-	IsDirect       bool                `json:"is_direct"`
-	ChannelMembers []string            `json:"channel_members"`
+	User           string                `json:"user"`
+	Channel        string                `json:"channel"`
+	Message        string                `json:"message"`
+	Props          model.StringInterface `json:"props"`
+	CreateAt       int64                 `json:"create_at"`
+	Type           string                `json:"type"`
```

Comment:
Hmm I wonder why this was commented out 🤔

Reply:
I honestly couldn't find a reason, it's been commented out for 5 years. Maybe because there was no reason to use it yet?